Blockzilla.xcodeproj/project.pbxproj

```diff
@@ -1428,7 +1431,7 @@
 			files = (
 				742C99D41F3A3AD200717D69 /* Assets.xcassets in Resources */,
 				16938C0121010C2D00DCD489 /* InfoPlist.strings in Resources */,
-				A89766DA1F57DCA9008183C5 /* BuildFile in Resources */,
```

Comment:
I'm a little bit confused by why/where this change is coming from.

Reply:
Fixed